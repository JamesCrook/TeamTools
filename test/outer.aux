\relax 
\providecommand\hyper@newdestlabel[2]{}
\providecommand\HyperFirstAtBeginDocument{\AtBeginDocument}
\HyperFirstAtBeginDocument{\ifx\hyper@anchor\@undefined
\global\let\oldcontentsline\contentsline
\gdef\contentsline#1#2#3#4{\oldcontentsline{#1}{#2}{#3}}
\global\let\oldnewlabel\newlabel
\gdef\newlabel#1#2{\newlabelxx{#1}#2}
\gdef\newlabelxx#1#2#3#4#5#6{\oldnewlabel{#1}{{#2}{#3}}}
\AtEndDocument{\ifx\hyper@anchor\@undefined
\let\contentsline\oldcontentsline
\let\newlabel\oldnewlabel
\fi}
\fi}
\global\let\hyper@last\relax 
\gdef\HyperFirstAtBeginDocument#1{#1}
\providecommand\HyField@AuxAddToFields[1]{}
\providecommand\HyField@AuxAddToCoFields[2]{}
\@writefile{toc}{\contentsline {chapter}{\numberline {1}New features in this release}{1}{chapter.1}}
\@writefile{lof}{\addvspace {10\p@ }}
\@writefile{lot}{\addvspace {10\p@ }}
\newlabel{f0}{{1}{1}{New features in this release}{chapter.1}{}}
\@writefile{toc}{\contentsline {section}{\numberline {1.1} New Logo }{1}{section.1.1}}
\@writefile{toc}{\contentsline {section}{\numberline {1.2} Themes\footnote {See URL http://alphamanual.audacityteam.org/man/Themes} }{1}{section.1.2}}
\@writefile{toc}{\contentsline {section}{\numberline {1.3} MIDI (and Allegro) Playback }{1}{section.1.3}}
\@writefile{toc}{\contentsline {section}{\numberline {1.4} Stem Plots }{1}{section.1.4}}
\@writefile{toc}{\contentsline {section}{\numberline {1.5} Menu reorganization }{2}{section.1.5}}
\@writefile{toc}{\contentsline {section}{\numberline {1.6} The Extended Menu bar }{2}{section.1.6}}
\@writefile{toc}{\contentsline {section}{\numberline {1.7} Appended recording on the same track in now the default }{2}{section.1.7}}
\@writefile{toc}{\contentsline {subsection}{\numberline {1.7.1} To record on a new track }{2}{subsection.1.7.1}}
\@writefile{toc}{\contentsline {section}{\numberline {1.8} Help buttons }{2}{section.1.8}}
\@writefile{toc}{\contentsline {section}{\numberline {1.9} Standard and Full shortcut sets }{3}{section.1.9}}
\@writefile{toc}{\contentsline {section}{\numberline {1.10} Selection Toolbar improvements }{3}{section.1.10}}
\@writefile{toc}{\contentsline {section}{\numberline {1.11} New commands for using clips via the keyboard }{3}{section.1.11}}
\@writefile{toc}{\contentsline {section}{\numberline {1.12} Running out of disk space }{3}{section.1.12}}
\@writefile{toc}{\contentsline {section}{\numberline {1.13} Additional new features\footnote {See URL http://alphamanual.audacityteam.org/man/New.features.in.this.release.-.appendix} }{3}{section.1.13}}
