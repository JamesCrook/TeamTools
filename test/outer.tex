\documentclass[twocolumn]{book}
\usepackage{blindtext}
\usepackage{enumitem}
\usepackage{graphicx}

    \usepackage{amsmath} 
    \usepackage[utf8]{inputenc} 
    \usepackage[T1]{fontenc} 

\usepackage{hyperref,stackengine}
\hypersetup{ colorlinks, citecolor=green, filecolor=blue, linkcolor=blue, urlcolor=blue } 
\usepackage[margin=0.6in]{geometry}
\setlength{\columnsep}{0.4 in}
\begin{document}
%%\newgeometry{oneside}
%%\cleardoublepage
{\let\cleardoublepage\clearpage 
\title{Audacity User Manual}
\author{James Crook}
%%\maketitle
}

\input{test.tex}


\end{document}