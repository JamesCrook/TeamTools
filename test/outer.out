\BOOKMARK [0][-]{chapter.1}{New features in this release}{}% 1
\BOOKMARK [1][-]{section.1.1}{ New Logo }{chapter.1}% 2
\BOOKMARK [1][-]{section.1.2}{ ThemesSee URL http://alphamanual.audacityteam.org/man/Themes }{chapter.1}% 3
\BOOKMARK [1][-]{section.1.3}{ MIDI \(and Allegro\) Playback }{chapter.1}% 4
\BOOKMARK [1][-]{section.1.4}{ Stem Plots }{chapter.1}% 5
\BOOKMARK [1][-]{section.1.5}{ Menu reorganization }{chapter.1}% 6
\BOOKMARK [1][-]{section.1.6}{ The Extended Menu bar }{chapter.1}% 7
\BOOKMARK [1][-]{section.1.7}{ Appended recording on the same track in now the default }{chapter.1}% 8
\BOOKMARK [2][-]{subsection.1.7.1}{ To record on a new track }{section.1.7}% 9
\BOOKMARK [1][-]{section.1.8}{ Help buttons }{chapter.1}% 10
\BOOKMARK [1][-]{section.1.9}{ Standard and Full shortcut sets }{chapter.1}% 11
\BOOKMARK [1][-]{section.1.10}{ Selection Toolbar improvements }{chapter.1}% 12
\BOOKMARK [1][-]{section.1.11}{ New commands for using clips via the keyboard }{chapter.1}% 13
\BOOKMARK [1][-]{section.1.12}{ Running out of disk space }{chapter.1}% 14
\BOOKMARK [1][-]{section.1.13}{ Additional new featuresSee URL http://alphamanual.audacityteam.org/man/New.features.in.this.release.-.appendix }{chapter.1}% 15
